packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -68,6 +68,8 @@ const DEFAULT_COLUMN_PROPERTIES: Partial<StarkTableColumnProperties> = {
 	isVisible: true
 };
 
+// FIXME: refactor the template of this component function to reduce its cyclomatic complexity
+/* tslint:disable:template-cyclomatic-complexity */
```

Comment:
Can you move these lines to the actual function that needs to be fixed? That way it is easier to spot where the problem is ;)

Reply:
@christophercr, actually the issue is in the html file, is it possible to use it there ?

Reply:
Oops, my bad.. I didn't read the `template` part in the rule name... so no worries, it is ok to leave it where you put it ;)

Reply:
oki ok nice